test/phone_form_field_test.dart

```diff
@@ -80,6 +82,49 @@ void main() {
         await tester.pumpWidget(getWidget(showFlagInInput: false));
         expect(find.byType(CircleFlag), findsNothing);
       });
+
+      testWidgets('Should format when shouldFormat is true', (tester) async {
+        PhoneNumber? phoneNumber = PhoneNumber.parse(
+          '',
+          destinationCountry: IsoCode.FR,
+        );
+
+        await tester.pumpWidget(
+            getWidget(initialValue: phoneNumber, shouldFormat: true));
+        await tester.pump(const Duration(seconds: 1));
+        final phoneField = find.byType(PhoneFormField);
+        await tester.enterText(phoneField, '677777777');
+        await tester.pump(const Duration(seconds: 1));
+        expect(find.text('6 77 77 77 77'), findsOneWidget);
+      });
+
+      testWidgets('Should show dial code', (tester) async {
```

Comment:
The tests are passing. Could you add "when showDialCode is true"  and "when showDialCode is false" ? Thanks.

Reply:
ok